media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -4431,8 +4431,194 @@ mos_gem_bo_is_exec_object_async(struct mos_linux_bo *bo)
     return bo_gem->exec_async;
 }
 
+/**
+ * enum intel_hwconfig - Global definition of hwconfig table attributes
+ *
+ * Intel devices provide a KLV (Key/Length/Value) table containing
```

Comment:
May I know the memory layout of Key, Length and Value? 

I am trying to understand the below logic. 

            assert(hw_info[i+1] == 1);
            gfx_info->SubSliceCount = hw_info[i+2];
            gfx_info->MaxSubSlicesSupported = hw_info[i+2];

Thanks a lot in advance!

Reply:
Hi Furong, the layout is,
K, L, V,
K, L, V,
...
For now, all the K/L/V are one unit32_t. So the L (hw_info[i+1]) is always 1.

Reply:
> 

Hi Furong, thanks for the review. The layout is,
K, L, V,
K, L, V,
...
For now, all the K/L/V are one unit32_t. So the L (hw_info[i+1]) is always 1.